src/main/java/com/conveyal/datatools/manager/controllers/api/ProjectController.java

```diff
@@ -120,10 +120,7 @@ private static Project updateProject(Request req, Response res) {
     private static Project deleteProject(Request req, Response res) {
         // Fetch project first to check permissions, and so we can return the deleted project after deletion.
         Project project = requestProjectById(req, "manage");
-        boolean successfullyDeleted = project.delete();
-        if (!successfullyDeleted) {
-            logMessageAndHalt(req, 500, "Did not delete project.", new Exception("Delete unsuccessful"));
-        }
+        project.delete();
```

Comment:
Just curious why this was changed.

Reply:
There's not a great reason. If I recall correctly, there were some merge conflicts related to the `Project#delete` method with the old `deploy-to-ec2` branch. I think my thinking was that this wouldn't be providing very valuable information about what went wrong (it was just returning a boolean from the mongo delete operation).